src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -258,7 +258,11 @@ public function setConstraintValidatorFactory(ConstraintValidatorFactoryInterfac
      */
     public function setTranslator(LegacyTranslatorInterface $translator)
     {
-        $this->translator = $translator instanceof LegacyTranslatorProxy ? $translator->getTranslator() : $translator;
+        $this->translator = $translator;
```

Comment:
@nicolas-grekas this code looks like sth from #31108

Reply:
oups indeed, fixed, thank!